tests/host.earth

```diff
@@ -16,6 +16,12 @@ add-ipv6:
     HOST cool.website dead:beef::cafe
     RUN cat /etc/hosts | grep "dead:beef::cafe\W*cool.website"
 
+expand-args:
+    ARG fake_host=example.com
+    ARG target_ip=1.2
+    HOST $fake_host $target_ip.3.4
+    RUN cat /etc/hosts | grep "1.2.3.4\W*example.com"
```

Comment:
we have a grep wrapper called [`acbgrep`](https://github.com/alexcb/acbtest?tab=readme-ov-file#acbgrep) which prints out failures messages.
```suggestion
    RUN cat /etc/hosts | acbgrep "1.2.3.4\W*example.com"
```